### PR TITLE
[Docs] Explain SGX v1 committed-RAM behavior for `sgx.enclave_size`

### LINKED ISSUE #233: Error in Pytorch.manifest: toml.decoder.TomlDecodeError: Duplicate keys!
Hi,

I am trying to run a new example in gramine running on Azure VM with PyTorch. I am using the following python.manifest.template. I followed the manifest template of pytorch in the graphene git (https://github.com/gramineproject/graphene/blob/master/Examples/pytorch/pytorch.manifest.template)

This is the manifest I am trying to run:

# PyTorch manifest template

loader.preload = "file:{{ gramine.libos }}"
libos.entrypoint = "{{ entrypoint }}"
loader.log_level = "{{ log_level }}"

loader.env.LD_LIBRARY_PATH = "/lib:/usr/lib:{{ arch_libdir }}:/usr/{{ arch_libdir }}"

loader.insecure__use_cmdline_argv = true
loader.insecure__use_host_env = true

loader.pal_internal_mem_size = "128M"

fs.mount.lib.type = "chroot"
fs.mount.lib.path = "/lib"
fs.mount.lib.uri = "file:{{ gramine.runtimedir() }}/"

fs.mount.lib2.type = "chroot"
fs.mount.lib2.path = "{{ arch_libdir }}"
fs.mount.lib2.uri = "file:{{ arch_libdir }}"

fs.mount.usr.type = "chroot"
fs.mount.usr.path = "/usr"
fs.mount.usr.uri = "file:/usr"

fs.mount.etc.type = "chroot"
fs.mount.etc.path = "/etc"
fs.mount.etc.uri = "file:/etc"

fs.mount.tmp.type = "chroot"
fs.mount.tmp.path = "/tmp"
fs.mount.tmp.uri = "file:/tmp"

fs.mount.pip.type = "chroot"
fs.mount.pip.path = "{{ env.HOME }}/.local/lib"
fs.mount.pip.uri = "file:{{ env.HOME }}/.local/lib"

sgx.nonpie_binary = true
sgx.enclave_size = "16G"
sgx.thread_num = 256

sgx.trusted_files = [
	"file:{{ entrypoint }}",
	"file:{{ gramine.runtimedir() }}/",
	"file:{{ arch_libdir }}/",
	"file:/usr/{{ arch_libdir }}/",
	"file:{{ python.stdlib }}/",
	"file:{{ python.distlib }}/",
	"file:{{ env.HOME }}/.local/lib/",
	"file:{{python.get_path('stdlib', vars={'installed_base': '/usr/local'}) }}",
	"file:scripts/",
    "file:/etc/mime.types",
    "file:/etc/default/apport",
 ]

sgx.allowed_files = [
 "file:/tmp",
 "file:/etc/apt/apt.conf.d",
 "file:/etc/apt/apt.conf",
 "file:/etc/default/apport",
 "file:/etc/nsswitch.conf",
 "file:/etc/group",
 "file:/etc/passwd",
 "file:/etc/host.conf",
 "file:/etc/hosts",
 "file:/etc/gai.conf",
 "file:/etc/resolv.conf",
 "file:/etc/fstab",
 "file:result.txt",
]

I was able to load gramine with sgx support successfully on my system. Tried the HelloWorld and Python examples from CI-Examples Folder.  Now I am trying to simple python scrip with gramine-sgx enabled

I get the following error when running make SGX =1
image

### PR DESCRIPTION
**Pull request by dimakuv**
## Description of the changes 

Many people encounter this situation, need to describe it somewhere in our docs.

Fixes #233 
Fixes https://github.com/gramineproject/gsc/issues/29

## How to test this PR? 

N/A.

---
This change is [Reviewable](https://reviewable.io/reviews/gramineproject/gramine/239)

### CODE REVIEW

**Review by mkow — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/239)* status: all files reviewed, 2 unresolved discussions, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel) (waiting on @dimakuv)

___
*[Documentation/manifest-syntax.rst, line 427 at r1](https://reviewable.io/reviews/gramineproject/gramine/239#-Mp6ldjB39WYVSqCm7yu:-Mp6ldjC8cxQDXTX4L7X:b-vwhenk) ([raw file](https://github.com/gramineproject/gramine/blob/d8876e371128f5a1942cd03a85562feb434816c5/Documentation/manifest-syntax.rst#L427)):*
> ```
> (RAM)
> ```

Not precisely, it's RAM+swap which is the limit. Ditto below.
___
*[Documentation/manifest-syntax.rst, line 440 at r1](https://reviewable.io/reviews/gramineproject/gramine/239#-Mp6lxSE8ul5PVd9B3z2:-Mp6lxSE8ul5PVd9B3z3:b-g4vmih) ([raw file](https://github.com/gramineproject/gramine/blob/d8876e371128f5a1942cd03a85562feb434816c5/Documentation/manifest-syntax.rst#L440)):*
> ```
>   Note that the swap file significantly decreases performance of the system when used!
> ```

At first I read this as "enabling swap makes your system slower", but I think you meant that if the app uses swap then it's slower? I'd reword this sentence to make it clear.

**Review by dimakuv — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/239)* status: 0 of 1 files reviewed, 2 unresolved discussions, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel), "fixup! " found in commit messages' one-liners (waiting on @mkow)

___
*[Documentation/manifest-syntax.rst, line 427 at r1](https://reviewable.io/reviews/gramineproject/gramine/239#-Mp6ldjB39WYVSqCm7yu:-Mp6nYsy5NqYG0gzq-EH:b-896fix) ([raw file](https://github.com/gramineproject/gramine/blob/d8876e371128f5a1942cd03a85562feb434816c5/Documentation/manifest-syntax.rst#L427)):*
<details><summary><i>Previously, mkow (Michał Kowalczyk) wrote…</i></summary><blockquote>

> ```
> (RAM)
> ```

Not precisely, it's RAM+swap which is the limit. Ditto below.
</blockquote></details>

Done.
___
*[Documentation/manifest-syntax.rst, line 440 at r1](https://reviewable.io/reviews/gramineproject/gramine/239#-Mp6lxSE8ul5PVd9B3z2:-Mp6nlWc0O5CNG5Kk1Hb:b-896fix) ([raw file](https://github.com/gramineproject/gramine/blob/d8876e371128f5a1942cd03a85562feb434816c5/Documentation/manifest-syntax.rst#L440)):*
<details><summary><i>Previously, mkow (Michał Kowalczyk) wrote…</i></summary><blockquote>

> ```
>   Note that the swap file significantly decreases performance of the system when used!
> ```

At first I read this as "enabling swap makes your system slower", but I think you meant that if the app uses swap then it's slower? I'd reword this sentence to make it clear.
</blockquote></details>

Done.

**Review by mkow — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/239)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (1 more required), "fixup! " found in commit messages' one-liners

___
*[Documentation/manifest-syntax.rst, line 440 at r1](https://reviewable.io/reviews/gramineproject/gramine/239#-Mp6lxSE8ul5PVd9B3z2:-Mp6rKzV3fPpiihFFTW1:b-tv4x6t) ([raw file](https://github.com/gramineproject/gramine/blob/d8876e371128f5a1942cd03a85562feb434816c5/Documentation/manifest-syntax.rst#L440)):*
<details><summary><i>Previously, dimakuv (Dmitrii Kuvaiskii) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Much better now :)

**Review by boryspoplawski — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/239)* status: all files reviewed, all discussions resolved, "fixup! " found in commit messages' one-liners

**Comment by dimakuv** [comment]
Jenkins, retest Jenkins-Direct-Sanitizers please (`test_libos.TC_30_Syscall.test_051_mmap_sgx` failed with `AssertionError: 'mmap test 5 passed' not found`, unrelated)

**Review by boryspoplawski — APPROVED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/239)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (1 more required)

**Review by mkow — APPROVED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/239)* status: :shipit: complete! all files reviewed, all discussions resolved
